desktops/kde.txt

```diff
@@ -0,0 +1,437 @@
+    |   $ wget $url/qt5-declarative.$qtver.tar.gz \                            |
+    |         -O qt5-declarative\#$qtver.tar.gz                                |
+    |   $ wget $url/qt5-webengine.$qtver.tar.gz \                              |
+    |          -O qt5-webengine\#$qtver.tar.gz                                 |
+    |                                                                          |
+    |   $ kiss i qt5                                                           |
+    |   $ kiss i qt5-declarative                                               |
+    |   $ kiss i qt5-webengine                                                 |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
+
+
+    [2.2] Installing
+    ____________________________________________________________________________
+
+    Now that all of the build requirements are taken care of, we can install 
+    the desktop environment. Make your choice between plasma-desktop or plasma, 
+    and build them by
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ kiss b plasma-desktop                                                | 
+    |   $ kiss i plasma-desktop                                                | 
+    |                                                                          |
+    +--------------------------------------------------------------------------+
+
+    Take care of any circular dependencies:
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ kiss b elogind                                                       |
+    |   $ kiss i elogind                                                       |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
+
+    Optionally, enable and start the eudev and dbus services:
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ ln -s /etc/sv/dbus  /var/service                                     |
+    |   $ ln -s /etc/sv/udevd /var/service                                     |
+    |                                                                          |
+    |   $ sv up dbus                                                           |
+    |   $ sv up udevd                                                          |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
```

Comment:
Enabeling theses services is also mentioned in the paragraph `login manager`.  
Maybe only one paragraph can be about which services are required for KDE and how to set them up.

Reply:
These specific services are only mentioned once; the ones listed later are for `sddm` to run, and neither `dbus` nor `eudev` are required for that. 
A services section seems overkill; the reasons for running these services is perhaps less self-evident than I think they are, but no services are actually required to run Plasma.

Reply:
My bad. I mixed that up with polkitd, elogind,... .  :see_no_evil: 
That no services are required to run `plasma` is a valueable information!

Reply:
I've included a small note about service requirements!